packages/kbn-interpreter/server/server_registries.js

```diff
@@ -56,7 +58,12 @@ export const populateServerRegistries = types => {
         require(path);
       });
 
-      global.canvas = undefined;
+      Object.keys(global).forEach(key => {
```

Comment:
Why? Come up with a new namespace instead of just blanket removing everything on global that wasn't there before `loadType` was called.

Reply:
unit tests will throw if we polute the global namespace, which happens here (with many different objects, thats why i aplied this general approach to remove everything of it, that was not there before.

Reply:
@ppisljar This caused https://github.com/elastic/kibana/issues/27162

The `regeneratorRuntime` was stripped from global, causing any `async` function that was transpiled to fail.

@rashidkpc's comment was pretty prophetic here.